packages/protocol/scripts/bash/generate-old-devchain-and-build.sh

```diff
@@ -35,10 +35,14 @@ git checkout $BRANCH 2>>$LOG_FILE >> $LOG_FILE
 
 echo "- Build monorepo (contract artifacts, migrations, + all dependencies)"
 cd ../..
-# TODO: use `yarn clean` after release v8 (not available at monorepo-root for <=v8)
-yarn run lerna run clean >> $LOG_FILE
+# XXX (soloseng): had to move the following command after `yarn reset` due to lerna incompatibility betwne node versions.
+# yarn run lerna run clean >> $LOG_FILE 
+# XXX (soloseng): use `yarn reset` to remove node18 node_modules before building with node12. 
+yarn run reset >> $LOG_FILE
 # build entire monorepo to account for any required dependencies.
 yarn install >> $LOG_FILE
+yarn run clean >> $LOG_FILE
+yarn install >> $LOG_FILE
```

Comment:
Having to use `yarn reset` to remove node18 node_modules before building with node12. 

This is due to lerna incompatibility between the different node versions, preventing `yarn run clean` from being executed before `yarn reset`.